skbio/stats/composition.py

```diff
@@ -708,6 +715,10 @@ def ancom(table, grouping,
 
         `"reject"` indicates if feature is significantly different or not.
 
+    pd.DataFrame
```

Comment:
Does this render correctly? I'm unsure about multiple returns formatted this way.